src/Node/DelimitedList.php

```diff
@@ -31,4 +31,17 @@ public function addToken($node) {
         }
         $this->children[] = $node;
     }
+
+    /**
+     * @return Node[]
+     */
+    public function getListElements() {
+        $listElements = [];
+        foreach ($this->children as $child) {
+            if($child instanceof Node) {
+                $listElements[] = $child;
```

Comment:
I noticed a `// TODO consider allowing empty delimiter to be more tolerant` in `parseDelimitedList`, so if allow insertion of empty delimiters (if there are none), the `getListElements` can return even key values of children. 
If this does sound okay to you, how would you prefer to represent these empty delimiters?